### DISCUSSION TITLE
Impossible de me connecter en SSH depuis la dernière MAJ système.

### DISCUSSION THREAD

**Discussion by SckyzO** [Guilouz/Klipper-Flsun-Speeder-Pad#159] [General]
Bonjour, 

Est ce que je suis le seul dans ce cas la ? 
J'ai mis à jour les packages via apt, aucun soucis, sauf que depuis il m'est impossible de me connecter en SSH pour l'utiliser "pi". 
J'ai le prompt qui demande le mot de passe, mais dès que je le rentre je perds la main, enfin plutôt le prompt ne me la rend pas. Comme si l'utilisateur n'avait pas de bash. Je n'ai pas envie de réinstaller ma machine :) 
Sinon j'ai bien Klipper qui fonctionne tout niquel, mais aucune connexion SSH possible. C'est embêtant ... Je cherche des idées si qqn en a je suis preneur.

**Comment by Guilouz** [comment]
De quelle maj tu parle?
De l’update 1.4 de Flsun? Si oui c’est normal il n’y a plus d’accès SSH avec cette version.

**Comment by SckyzO** [comment]
Non du tout. Je suis sur une flsun rootée
